### PR TITLE
Update plugin URLs and remove broken plugin entries

### PR DESCRIPTION
**Pull request by coderstephen**
A number of plugins in the package database have moved repository locations, or have simply been deleted. This PR updates the URLs of any plugins to their new GitHub locations, and removes entries to plugins that are no longer available.

### CODE REVIEW

File: db/themes/pure [hunk 1]
```diff
@@ -1 +1 @@
-https://github.com/rafaelrinaldi/theme-pure
+https://github.com/rafaelrinaldi/pure
```
Comment (oranja) [inline db/themes/pure]: We still need the `theme-` prefix for install, IIRC

Comment (bobthecow) [inline db/themes/pure]: Right. The redirect there was a favor to OMF. Let's hold off on breaking our packages until we support prefixless repo names :)

Comment (derekstavis) [inline db/themes/pure]: +1 for keeping the `theme-` prefix. Unfortunately it's still needed.

**Comment by coderstephen** [comment]
Sorry about the delay. Just put the `theme-` prefix back.

**Comment by derekstavis** [comment]
I think this can be merged. Rebase?

**Comment by coderstephen** [comment]
Literally just did. :+1:

**Comment by derekstavis** [comment]
That was fast :)

**Pull request merged**